installing/installing_disconnected/installing-disconnected-aws.adoc

```diff
@@ -0,0 +1,126 @@
+[id="installing-disconnected-aws"]
+= Installing a cluster on AWS that uses mirrored installation content
+include::modules/common-attributes.adoc[]
+:context: installing-disconnected-aws
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a
+cluster on Amazon Web Services (AWS) using infrastructure that you provide and
+an internal mirror of the installation release content.
+
+[IMPORTANT]
+====
+While you can install a {product-title} cluster by using mirrored installation
+release content, your cluster still requires internet access to use the AWS APIs.
+====
+
+One way to create this infrastructure is to use the provided
+CloudFormation templates. You can modify the templates to customize your
+infrastructure or use the information that they contain to create AWS objects
+according to your company's policies.
+
+.Prerequisites
+
+//* xref:../../installing/installing_disconnected/installing-disconnected-preparations.adoc[Create a mirror registry on your bastion host]
+// and obtain the `imageContentSources` data for your version of {product-title}.
+* Review details about the
+xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update]
+processes.
+* xref:../../installing/installing_aws/installing-aws-account.adoc#installing-aws-account[Configure an AWS account]
+to host the cluster.
++
+[IMPORTANT]
+====
+If you have an AWS profile stored on your computer, it must not use a temporary
+session token that you generated while using a multi-factor authentication
+device. The cluster continues to use your current AWS credentials to
+create AWS resources for the entire life of the cluster, so you must
+use key-based, long-lived credentials. To generate appropriate keys, see
+link:https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html[Managing Access Keys for IAM Users]
+in the AWS documentation. You can supply the keys when you run the installation
+program.
+====
+* Download the AWS CLI and install it on your computer. See
+link:https://docs.aws.amazon.com/cli/latest/userguide/install-bundle.html[Install the AWS CLI Using the Bundled Installer (Linux, macOS, or Unix)]
+in the AWS documentation.
+* If you use a firewall, you must
+xref:../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configure it to access Red Hat Insights].
+
+include::modules/cluster-entitlements.adoc[leveloffset=+1]
+
+include::modules/installation-aws-user-infra-requirements.adoc[leveloffset=+1]
+
+include::modules/installation-aws-permissions.adoc[leveloffset=+2]
+
+include::modules/installation-obtaining-installer.adoc[leveloffset=+1]
+
+include::modules/ssh-agent-using.adoc[leveloffset=+1]
+
+include::modules/installation-generate-aws-user-infra.adoc[leveloffset=+1]
+
+// After the proxy change merges, I need to put it in and emphasize that you
+// must configure a proxy for the AWS mirrored content story.
```

Comment:
You *can* use a proxy.  I don't think you *must* use a proxy unless you close off enough network access that we need it to access the AWS IAM endpoints or something.

Reply:
You're right, and I'm going to clarify the AWS proxy requirements and usage on https://bugzilla.redhat.com/show_bug.cgi?id=1743483